wiki/Gameplay/Spinner_trap/en.md

```diff
@@ -0,0 +1,7 @@
+# Spinner Trap
+
+A term used in [osu!catch](/wiki/Game_mode/osu!catch) when the [fruit](/wiki/Gameplay/Hit_object/Fruit) after a [spinner](/wiki/Gameplay/Hit_object/Spinner) is not easily accessible naturally. Several ways can cause this:
+
+- The follow-up fruit is on the outer edges of the playfield.
+- The follow-up fruit is placed at an unreachable distance from the last banana in the optimal spin path.
+- The follow-up fruit being obfuscated by [bananas](/wiki/Gameplay/Hit_object/Banana).
```

Comment:
I agree that adding a picture to this would be helpful. However, I would rather wait for lazer to be more in a ready state so we can add patterns showcasing this in the new skin instead of using the old stable one.

But defiantly something to improve, not to make this article a stub.